wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/podcasts/PodcastsScreen.kt

```diff
@@ -31,9 +40,17 @@ fun PodcastsScreen(
 
     ScalingLazyColumn(
         modifier = modifier.fillMaxWidth(),
-        state = listState,
+        state = listState
```

Comment:
Curious how you feel about trailing commas. I like them because they make the diffs smaller when adding/removing items, but it looks like you may not be a fan? Obviously this is a SUPER CRITICAL ISSUE! :trollface: 

I'll actually look into seeing if we can get spotless to enforce this one way or the other (although I like trailing commas, I'm fine having spotless enforce either approach).

Reply:
It would be good to enforce this. I don't mind either way which might show by the way I do both. 😁